test/boolattrs.ts

```diff
@@ -0,0 +1,32 @@
+"use strict";
+
+// Boolean HTML attributes, in the case React expects.
+const booleanAttrs = [
+  "allowFullScreen",
```

Comment:
Why are you diverging from the project's style of using single quotes? Could you revert to single quotes instead, so we avoid this intrusive change?